docs/stress-testing.md

```diff
@@ -37,8 +37,14 @@ You can
 use [this Tab Auto Refresh Chrome extension](https://chrome.google.com/webstore/detail/tab-auto-refresh/oomoeacogjkolheacgdkkkhbjipaomkn)
 to auto-refresh the page.
 
-Open `test/stress/main.go` and run it with a large number (e.g. 10000).
+Run `go run ./test/stress/tool -n 10000` to run a large number of workflows.
 
-* Use Prometheus to analyse this.
-* Use PProf using `./hack/capture-pprof.sh` and look at allocs and profile for costly areas.
+* Use Prometheus to see how many Kubernetes API requests are being made.
```

Comment:
Would you like to include some quantitative measures here? How many requests are too many?